modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/DDMFormPortlet.java

```diff
@@ -164,8 +164,10 @@ public void render(
 					renderRequest, ddmFormDisplayContext);
 			}
 
-			if (ddmFormDisplayContext.isRequireAuthentication() &&
-				ddmFormDisplayContext.isSharedURL()) {
+			if ((ddmFormDisplayContext.isRequireAuthentication() &&
+				 ddmFormDisplayContext.isSharedURL()) ||
+				(ddmFormDisplayContext.isLimitToOneSubmissionPerUserEnabled() &&
+				 !ddmFormDisplayContext.isLoggedUser())) {
```

Comment:
@marcelabc can you reorder this by putting `isLimitToOneSubmissionPerUserEnabled` and `isLoggedUser` before `isRequireAuthentication`? Thanks!